kyo-core/shared/src/main/scala/kyo/typeMap.scala

```diff
@@ -10,7 +10,19 @@ extension [A](self: TypeMap[A])
         throw new RuntimeException(s"fatal: kyo.TypeMap of contents [$self] missing value of type: [${t.show}].")
 
     inline def get[B >: A](using inline t: Tag[B]): B =
```

Comment:
@fwbrasil do you think this is still worth inlining now?

Reply:
Probably not given the method size. There isn't much benefit as well since the implementation doesn't have calls that could become monomorphic if inlined.